templates/mobile/activation/activation.1.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "activation",
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "description": "hashed version of Google ID"
+    },
+    "client_id": {
+      "type": "string",
+      @COMMON_PATTERN_UUID_1_JSON@,
+      "description": "telemetry client_id, included only if identifer is empty"
+    },
+    "manufacturer": {
+      "type": "string",
+      "description": "Build.MANUFACTURER"
+    },
+    "model": {
+      "type": "string",
+      "description": "Build.MODEL"
+    },
+    "device": {
+      "type": "string"
+    },
+    "distribution_id": {
+      "type": "string"
+    },
+    "locale": {
+      "type": "string",
+      "description": "application locale, such as 'en-US'"
+    },
+    "os": {
+      "type": "string"
+    },
+    "osversion": {
```

Comment:
We do have some special handling in the GCP pipeline to find core-ping-style "os" and "osversion" fields, so we leave this name be rather than changing it to "os_version".